vxfreeswitch/voice.py

```diff
@@ -349,6 +356,9 @@ def send_inbound_message(self, client, text, session_event):
             content=text,
             transport_name=self.transport_name,
             transport_type=self._transport_type,
+            helper_metadata={
+                'call_duration': duration
```

Comment:
I wonder if we should only include the `call_duration` in the message that ends the session and whether it should be inside a `voice` sub-section in helper_metadata instead of at the top-level?